README.md

```diff
@@ -89,15 +89,18 @@ In addition to template rules, SonarDelphi can be extended with custom rules plu
 >
 > SonarDelphi requires source code for all dependencies, including the standard library.
 
-## Configuration
+## Read the Manual
 
-You can configure SonarDelphi [analysis parameters](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/analysis-parameters/)
-on the SonarQube server at **Administration > General Settings > Delphi**.
+> [!TIP]
```

Comment:
Drat, this is now repetitive with the adjacent content, since we immediately talk about the manual and also immediately mention how it goes over the basics of the Sonar ecosystem.

A couple options here:
- remove the tip
- if we're married to having a call-out, reword the tip

> [!TIP]
> Are you a first-time SonarQube user?
> Read the [SonarQube from first principles](docs/MANUAL.md#sonarqube-from-first-principles) section.

Reply:
I mean, the SonarQube from first principles section is the first section of the manual. If they look at the manual they'll read it, so this callout is really of limited use under the "Read the Manual" heading.

Thoughts on adding it to the end of the quickstart? e.g.

> 5. Learn more:
>    * Read about [SonarQube from first principles](#)
>    * [Read the manual](#)

or similar?

Reply:
I've moved the tip somewhere else, see what you think.